docroot/modules/humsci/hs_field_helpers/templates/views-view-pattern.html.twig

```diff
@@ -1,3 +1,3 @@
-<div{{ attributes }}>
+<div{{ attributes.addClass('views-view-pattern') }}>
```

Comment:
I have found with earth that this file had to be available in a module as well.

Reply:
I recently found that the "pattern" style only correctly works with a pattern row plugin. If i try to use just fields, it doesn't wrap each row in a div. which broke when i tried to do a masonry pattern.
I'll have to take a look at that more later to step trough for that scenario